<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIShareBrowserSearch</class>
 <widget class="QDialog" name="UIShareBrowserSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_SEARCHSTR"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_SEARCH">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_TYPE_SEARCH">
     <item>
      <property name="text">
       <string>Search only files</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Search only directories</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Search all</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Directory</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_STATS">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
